### PR TITLE
Add accident severity filter to heat map

### PR DESCRIPTION
**Pull request by shakedzy**

### CODE REVIEW

File: anyway/infographics_utils.py [hunk 1]
```diff
@@ -647,12 +647,14 @@ def create_infographics_data(news_flash_id, number_of_years_ago):
     output["widgets"].append(accident_count_by_accident_type.serialize())
 
     # accidents heat map
+    accidents_heat_map_filters = location_info
+    accidents_heat_map_filters['accident_severity'] = [1, 2]
```
Comment (BarVolunteering) [inline anyway/infographics_utils.py:651]: Can you please change the numbers 1,2 to consts of the accident severity

**Review by BarVolunteering — CHANGES_REQUESTED** [review]

**Comment by atalyaalon** [comment]
@shakedzy did you test this?

**Comment by atalyaalon** [comment]
@shakedzy looks very good BTW :)

**Review by atalyaalon — COMMENTED** [review]
Looks good to me, just the the consts to the BE constants file https://github.com/hasadna/anyway/blob/dev/anyway/backend_constants.py
(You can use ACCIDENT_SEVERITY_DEADLY const and add another for SEVERE)

**Comment by shakedzy** [comment]
@atalyaalon - yes, I tested it. Replaced numbers with constants.

**Pull request merged**